alembic/env.py

```diff
@@ -0,0 +1,78 @@
+from logging.config import fileConfig
+from sqlalchemy import engine_from_config, pool
+from alembic import context
+from database import Base
```

Comment:
Ensure that the `Base` class is correctly imported from the `database` module. It should be the declarative base for your SQLAlchemy models.